File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest34891.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import java.nio.file.Files;
import java.nio.file.Paths;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest34891 {

    @PostMapping(path="/BenchmarkTest34891", consumes="multipart/form-data")
    public void BenchmarkTest34891(@RequestPart("multipart_field") String multipartField, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String multipartValue = multipartField != null ? multipartField : "";
        Files.write(Paths.get("/var/uploads/" + multipartValue), "data".getBytes());
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}

File: Benchmarks/enterprise/python/django/testcode/benchmark_test_34891.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest34891(request):
    raw_body = request.body.decode('utf-8')
    data = f'{raw_body}'
    int(str(data))
    return JsonResponse({"saved": True})
